subsys/net/ip/rpl.c

```diff
@@ -3286,6 +3286,24 @@ static int forwarding_dao(struct net_rpl_instance *instance,
 	return r;
 }
 
+static bool is_root(struct net_rpl_instance *instance)
+{
+	if (instance && instance->current_dag) {
```

Comment:
you could have the reverse test to return false directly here.

It would be easier to read, and avoid to break the line on the if () condition below.

Just a taste preference though.

Btw, (it's weird we can't comment the commit message): any rfc reference to add in commit message?